tests/Unit/Routes/StoreRoutesTest.php

```diff
@@ -386,7 +388,7 @@ public function testCentreRegistrationsSummaryGate()
             "centre_id" => $this->centre->id,
         ]);
 
-        $route = $specificCentreRoute = URL::route('store.centre.registrations.summary', ['centre' => $this->centre->id]);
```

Comment:
cant recall why i had it set that way :thinking:

Reply:
don't know 🤔 but removing it doesn't break anything. there's also an unused declaration `use Illuminate\Support\Collection;` that we can remove from this file, I think.

Reply:
i'm using that to specifically highlight $registrations as collection; on like 21.
It helps with autocomplete in IDE's
Can be removed.